apps/server/src/modules/room/repo/entity/room.entity.ts

```diff
@@ -0,0 +1,40 @@
+import { Entity, Property } from '@mikro-orm/core';
+import { BaseEntityWithTimestamps } from '@shared/domain/entity/base.entity';
+import { Room } from '../../domain/do/room.do';
+
+export interface RoomEntityProps {
+	id?: string;
+	name: string;
+	color: string;
+	startDate?: Date;
+	untilDate?: Date;
+}
+
+@Entity({ tableName: 'rooms' })
+export class RoomEntity extends BaseEntityWithTimestamps {
+	@Property()
+	name: string;
+
+	@Property()
+	color: string;
+
+	@Property()
```

Comment:
`@Property({ nullable: true })` here ? or is that not necessary?

Reply:
`{ nullable: true }` is the MikroORM default.